definitions/artifacts/cryptographic-keypair.json

```diff
@@ -0,0 +1,62 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema",
+  "$md": {
+    "access": "public",
+    "name": "keypair"
+  },
+  "type": "object",
+  "title": "Keypair",
```

Comment:
Title should be human version of file name. This will be displayed in the UI like on bundle params